_shows/21_22/scottish_falsetto_sock_puppet_theatre.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Scottish Falsetto Sock Puppet Theatre
+period: Spring
```

Comment:
Is there a company for this show?

Reply:
i think the title and company are the same? would i need to write it twice?